ui-admin/src/portal/siteContent/SiteContentEditor.tsx

```diff
@@ -285,44 +331,24 @@ const SiteContentEditor = (props: InitializedSiteContentViewProps) => {
                 setSelectedNavOpt(e ?? landingPageOption)
               }}/>
           </div>
-          { portalEnvContext.portalEnv.supportedLanguages.length > 1 && <div className="ms-2" style={{ width: 200 }}>
-            <Select options={languageOptions} value={selectedLanguageOption} inputId={selectLanguageInputId}
-              isDisabled={hasInvalidSection} aria-label={'Select a language'}
-              onChange={languageOnChange}/>
-          </div> }
           <Button className="btn btn-secondary"
             tooltip={'Add a new page'}
             disabled={readOnly || !isEditable || hasInvalidSection}
             onClick={() => setShowAddPageModal(!showAddPageModal)}>
-            <FontAwesomeIcon icon={faPlus}/> Add navbar item
+            <FontAwesomeIcon icon={faPlus}/> Add
```

Comment:
Maybe it's just me, but I think I would be confused with just `Add`/`Delete`. I think I slightly prefer it still being `Add navbar item`. But I really like that it's closer to the select.

Reply:
yeah, I could see this going either way -- I'll see what people think at demo